app/src/main/java/com/google/android/stardroid/control/RealClock.java

```diff
@@ -22,7 +26,10 @@
 public class RealClock implements Clock {
   @Override
   public long getTimeInMillisSinceEpoch() {
-    // TODO(johntaylor): consider using SystemClock class.
-    return System.currentTimeMillis();
+    Calendar currentTime = Calendar.getInstance();
+    // For getting the time in UTC
+    long offset = currentTime.get(Calendar.ZONE_OFFSET) +
+            currentTime.get(Calendar.DST_OFFSET);
+    return currentTime.getTimeInMillis() + offset;
```

Comment:
Why are the offsets needed? The API docs say getTimeInMillis already returns UTC.  I'm a bit rusty on this stuff.  It's not obvious to me why this is better than the existing code.

Reply:
I don't think it is - the existing code seems right to me.

Reply:
Well not that it is completely true. I have seen time differences using the `SystemClock` . The offsets are there for confirmation on the accuracy (if ever there is a time zone issue on any device) . @jaydeetay